networking/firewall/000infralib/metadata

```diff
@@ -0,0 +1,13 @@
+[General]
+name=Networking Firewall Library
+owner=Networking QE
+description=Helper functions for networking firewall tests
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=
+softDependencies=
```

Comment:
mentioned on previous comment,  we could  remove empty dependencies and softDependencies lines in the test metadata

Reply:
Removed.